waspc/data/Generator/templates/react-app/src/entities/_entity/components/CreateForm.js

```diff
@@ -64,8 +70,93 @@ export default class {= formName =} extends React.Component {
   }
 
   render() {
+    const form = (
+      <form noValidate onSubmit={this.handleSubmit} action="javascript:void(0);">
+        {=# formFields =}
+        {=# boolean =}
+        {=# show =}
+        <div>
+          <FormControlLabel
+            {=# label =}
+            label="{= label =}"
+            {=/ label =}
+            control={
+              <Switch
+                checked={this.getField('{= name =}')}
+                onChange={() => this.toggleField('{= name =}')}
+                value="{= name =}"
+              />
+            }
+          />
+        </div>
+        {=/ show =}
+        {=/ boolean =}
+        {=# string =}
+        {=# show =}
+        <div>
+          <TextField
+            {=# label =}
+            label="{= label =}"
+            {=/ label =}
+            {=# placeholder =}
+            placeholder="{= placeholder =}"
+            {=/ placeholder =}
+            value={this.getField('{= name =}')}
+            onChange={event => this.setField('{= name =}', event.target.value)}
+            margin="normal"
+            fullWidth
+            InputLabelProps={{
+              shrink: true
+            }}
+          />
+        </div>
+        {=/ show =}
+        {=/ string =}
+        {=/ formFields =}
+        {=# showSubmitButton =}
+        <div>
+          <Button type="submit" variant="contained" color="primary">
+            {this.props.submitButtonLabel || 'Submit'}
+          </Button>
+        </div>
+        {=/ showSubmitButton =}
+      </form>
+    )
+
+    {=# isXRayModeEnabled =}
+    const fields = {
+      {=# formFields =}
+        {= name =}: {
+          show: {=# show =}true{=/ show =}{=^ show =}false{=/ show =},
+
+          {=^ show =}
+          defaultValue: {= defaultValue =}
+          {=/ show =}
+        },
+      {=/ formFields =}
+    }
+
+    const entity = {
+      {=# formFields =}
+        {= name =}: '{= type =}',
+      {=/ formFields =}
+    }
+    {=/ isXRayModeEnabled =}
+
     return (
       <div className={this.props.className}>
+      {=# isXRayModeEnabled =}
+      <EntityFormInfoBox
+        isXRayModeOn={this.props.isXRayModeOn}
+        
+        entityName="{= entityClassName =}"
+        formName="{= formName =}"
+
+        entity={entity}
+        fields={fields}
+      >
+        <>
```

Comment:
```suggestion
        <>
```
I am not sure but I think you can remove this fragment?

Reply:
you are right, I removed it.